jupyter_book/book_template/Makefile

```diff
@@ -11,7 +11,10 @@ help:
 
 
 install:
-	gem install bundler
+	# Check to see whether bundler is already installed. If not, install it.
+	if [ hash bundler 2>/dev/null ]; then \
```

Comment:
Is this indentation slightly off? Shouldn't the `if` statement be at the same level as the `fi` statement? (apologies if this isn't Makefile convention, just double checking)

Reply:
nvm, this is just a github display quirk, the raw file looks good